src/controllers/engine/controllerengine.cpp

```diff
@@ -516,6 +516,12 @@ void ControllerEngine::errorDialogButton(
 
 ControlObjectScript* ControllerEngine::getControlObjectScript(const QString& group, const QString& name) {
     ConfigKey key = ConfigKey(group, name);
+
+    if (key.isNull()) {
```

Comment:
This is implemented as 
```
return group.isNull() && item.isNull();
```

I think you want tho check for 
```
group.isEmpty() || item.isEmpty()
```
In terms of script debugging, I think we should give an extra warning per parameter.

Reply:
Looking at the use cases for ConfigKey::isEmpty() and ConfigKey::isNull() 
I think we should dispose both and replace it with 
```
isInvalid() {
    group.isEmpty() || item.isEmpty()
}
```
 
or
``` 
isValid() {
    !group.isEmpty() && !item.isEmpty()
}
```

Reply:
Good catch!